plugins/database/redshift/redshift_test.go

```diff
@@ -512,12 +511,8 @@ ALTER USER "{{name}}" WITH PASSWORD '{{password}}';
 // helper file in the future.
 func createTestPGUser(t *testing.T, connURL string, username, password, query string) {
 	t.Helper()
-	conn, err := pq.ParseURL(connURL)
```

Comment:
Leaving it out of the code except for the test that checks DSN string compatibility.

> ParseURL no longer needs to be used by clients of this library since supplying a URL as a connection string to sql.Open() is now supported: